src/operator/rnn-inl.h

```diff
@@ -28,6 +28,7 @@
 
 #define MXNET_USE_CUDNN_RNN MXNET_USE_CUDNN == 1 && CUDNN_MAJOR >= 5
 #define USE_CUDNN_LSTM_PROJ MXNET_USE_CUDNN == 1 && CUDNN_VERSION >= 7200
+#define USE_VAR_SEQ_LENGTH MXNET_USE_CUDNN == 1 && CUDNN_VERSION >= 7200
```

Comment:
seems unnecessary for the duplicate

Reply:
Fair enough. At first I added it since I was putting things that had nothing to do with LSTM_PROJ under this #ifdef guard.

Maybe I'll just rename it to something that makes sense for both uses.